data/tooltips.txt

```diff
@@ -164,6 +164,15 @@ tip "hull heat multiplier:"
 tip "hull repair rate:"
 	`Amount of hull strength that can be repaired per second. Repairs usually consume energy and may generate heat as well.`
 
+tip "absolute threshold:"
+	`The remaining hull strength at which the ship becomes disabled, unaltered by any other attributes.`
+
+tip "hull threshold:"
+	`Alters the hull strength value at which the ship will become disabled.`
```

Comment:
Given that the "threshold percentage" mentions that it is a percentage, perhaps clarifying that this is an amount of hull and *not* a percentage might be good.

Reply:
I feel like this fact would be apparent in its usage, given that "threshold percentage" will always be a number between 0 and 1 (shown in game as 0 to 100%), whereas hull threshold will also be a number without any units that is likely to be >100 from the start.